src/app/api/community.service.ts

```diff
@@ -0,0 +1,98 @@
+import { Injectable } from '@angular/core';
+import { BaseApiService, Version } from './base-api.service';
+
+export interface Community {
+  name: string;
+  title: string;
+  description: string;
+  sidebar: string;
+  nsfw: boolean;
+  creatorId: number;
+}
+
+export interface CommunityList {
+  communities: Array<string>;
+  totalCount: number;
+  offset: number;
+  size: number;
+  sort: number;
+}
+
+export interface TemplateList {
+  templates: Array<string>;
```

Comment:
Since we don't have a template interface, we can use `templates: Array<{}>;` for now